### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by xxDark**

### CODE REVIEW

**Comment by xxDark** [comment]
> 
> 
> The capacity of the StringBuilders shouldn't be changed, it's there to prevent large unwanted data from being sent to the proxy.

On windows, malloc for example aligns the allocated memory to the size of the memory pag, and Java does the same thing - it reserves some space for realloc, so it makes no sence to set initial capacity for message with unknown length.

File: proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommandManager.java [hunk 1]
```diff
@@ -13,10 +13,13 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.Set;
+import java.util.regex.Pattern;
+
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 public class VelocityCommandManager implements CommandManager {
 
+  private static final Pattern SPLIT_PATTERN = Pattern.compile(" ");
```
Comment (Leymooo) [inline proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommandManager.java]: Revert this. String#split is faster. 
https://stackoverflow.com/a/14901754
https://github.com/SpigotMC/BungeeCord/pull/2539

Comment (lucko) [reply proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommandManager.java]: Guava's `Splitter` works too.
https://google.github.io/guava/releases/23.0/api/docs/com/google/common/base/Splitter.html

Comment (xxDark) [reply proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommandManager.java]: > 
> 
> Guava's `Splitter` works well too.
> https://google.github.io/guava/releases/23.0/api/docs/com/google/common/base/Splitter.html

Gonna make a quick JMH benchmark.

Comment (xxDark) [reply proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommandManager.java]: > 
> 
> > Guava's `Splitter` works well too.
> > https://google.github.io/guava/releases/23.0/api/docs/com/google/common/base/Splitter.html
> 
> Gonna make a quick JMH benchmark.

Yeah, Guava's `Splitter` is much faster then Java's one.

**Comment by xxDark** [comment]
> 
> 
> The capacity of the StringBuilders shouldn't be changed, it's there to prevent large unwanted data from being sent to the proxy.

Benchmark                    Mode  Cnt         Score        Error  Units
BuilderBenchmark.testDefault  thrpt   25  16446379,016 ± 635051,541  ops/s
BuilderBenchmark.testPredict  thrpt   25   2398699,209 ± 150939,535  ops/s

**Comment by astei** [comment]
Thanks!

**Comment by xxDark** [comment]
@astei please merge my latest commit to fix builds on Jenkis

**Comment by astei** [comment]
I fixed it, don't worry.

**Pull request merged**